dll/win32/syssetup/wizard.c

```diff
@@ -2968,6 +2970,29 @@ ProcessSetupInf(
     }
 #endif
 
+    *szValue = UNICODE_NULL;
+    pData = szValue;
+
+    GetWindowsDirectoryW(szValue, ARRAYSIZE(szValue));
+    StringCchCatW(szValue, ARRAYSIZE(szValue), L"\\inf");
+
+    if (FAILED(StringCchLengthW(szValue, ARRAYSIZE(szValue), &pcchLength)))
```

Comment:
Please rename `pcchLength` into `cchLength`, since this is a string length, not a pointer.